bin/build

```diff
@@ -45,18 +41,30 @@ if [ -z "$REPO" ]; then
   exit 1
 fi
 
+if [ -n "$LOCAL_RPM" ] && [ -n "$REBUILD_RPM" ]; then
+  echo "Local rpm (-l) and rebuild rpm (-b) options can't be used together"
+  exit 1
+fi
+
 set -e
 
+if [ -n "$REBUILD_RPM" ]; then
+  rpm_dir=$IMAGE_DIR/manageiq-base/rpms
+  rm -rf $rpm_dir/*
+  options="-v $PWD/$rpm_dir:/root/BUILD/rpms"
+  if [ -n "$RPM_BUILD_OPTIONS" ]; then
+    options+=" -v $PWD/$RPM_BUILD_OPTIONS:/root/OPTIONS"
```

Comment:
Why prefix with $PWD?  I would expect the caller to specify the proper path

Reply:
`-v` doesn't take relative path and it will fail to mount.

Reply:
If `realpath` is available, maybe that's preferable, but I'm good with this for now.

Reply:
I wasn't sure what command works on different platforms (linux, osx, etc), so I went for $PWD...

Reply:
yeah realpath isn't in mac by default, but it's a simple brew command away.